jtransc-gen-php/src/com/jtransc/gen/php/PhpTarget.kt

```diff
@@ -413,7 +413,7 @@ class PhpGenerator(injector: Injector) : CommonGenerator(injector) {
 
 	override fun getClassInterfaces(clazz: AstClass): List<FqName> = clazz.implementingNormalized
 
-	val AstType.targetNameNullable get() = if (this.isReference()) "?$targetName" else targetName
+	val AstType.targetNameNullable get() = if (this.isReference()) "&$targetName" else targetName
```

Comment:
I think this is for typehinting so ? was not a typo. & is for varables to create a reference in php. In this case reference means a java type by reference, classes and arrays (not a primitive). Jtransc requires php 7.1 (dont remember if this works on hhvm in the end).